server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoError/index.ts

```diff
@@ -0,0 +1,238 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BaseMongoExceptionRetryRule, IMongoExceptionRetryRule } from "../IMongoExceptionRetryRule";
+class InternalErrorRule extends BaseMongoExceptionRetryRule {
+    private static readonly codeName = "InternalError";
+    protected defaultRetryDecision: boolean = false;
+
+    constructor(retryRuleOverride: Map<string, boolean>) {
+        super("InternalErrorRule", retryRuleOverride);
```

Comment:
>InternalErrorRule [](http://example.com/codeflow?start=15&length=17)

(nit) do we want to use InternalErrorRule.name?

Reply:
No. The flection like things seems attempting, however, it will stop work if we minify/uglify js code:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name#javascript_compressors_and_minifiers